test/tools/ossfuzz/protoToAbiV2.h

```diff
@@ -389,6 +399,28 @@ class ProtoConverter
 		);
 	}
 
+	/// Accepts a monotonically increasing counter as input and
+	/// returns a value to be used to compute the length of a
+	/// hex or string literal. This function is called by
+	/// bytesArrayValueAsString().
+	static unsigned getVarLength(unsigned _counter)
+	{
+		return (_counter + 879) * 32 % (s_maxDynArrayLength + 1);
```

Comment:
All these magic numbers are a mistery to me ;)

Reply:
Yeah, this is hacky. I should have left a comment there.